### PR TITLE
Packaging for release v3.3.6

### PR DESCRIPTION
**Pull request by carlos-villavicencio-adsk**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -4,6 +4,16 @@ ShotGrid Python API Changelog
 
 Here you can see the full list of changes between each Python API release.
 
+v3.3.6 (TBD)
```
Comment (julien-lang) [inline HISTORY.rst:7]: TODO TBD

File: setup.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
 setup(
     name='shotgun_api3',
-    version='3.3.5',
+    version='3.3.6.dev',
```
Comment (julien-lang) [inline setup.py:30]: .dev to be removed

**Review by julien-lang — APPROVED** [review]
LGTM!

**Review by NorberMV — APPROVED** [review]
LGTM, good job!

**Pull request merged**